conda_build/skeletons/pypi.py

```diff
@@ -1218,14 +1219,17 @@ def run_setuppy(src_dir, temp_dir, python_version, extra_specs, config, setup_op
     :param temp_dir: Temporary directory for doing for storing pkginfo.yaml
     :type temp_dir: str
     '''
-    # TODO: we could make everyone's lives easier if we include packaging here, because setuptools
-    #    needs it in recent versions.  At time of writing, it is not a package in defaults, so this
-    #    actually breaks conda-build right now.  Omit it until packaging is on defaults.
```

Comment:
Shouldn't we add packaging here, now that it's part of defaults?

Reply:
Setuptools vendors it since 2014 but it's worth testing